app/domains/schulden/vorabcheck/context.ts

```diff
@@ -1,8 +1,26 @@
 import { z } from "zod";
-import { YesNoMaybeAnswer } from "~/services/validation/YesNoAnswer";
+import { YesNoAnswer } from "~/services/validation/YesNoAnswer";
+
+export const schuldenBei = z.enum([
+  "privatpersonen",
+  "behoerden",
+  "sparkasseNichtZurueckzahlen",
+  "gesetzlichenKrankenkasse",
+  "rechnungenNichtBezahlen",
+  "beitragsserviceGEZ",
+  "unterhaltsschulden",
+  "strafeNichtZahlen",
+  "finanzamt",
+  "hauptzollamt",
+  "nichtSagen",
+  "weissNicht",
+]);
```

Comment:
i would still shorten and further clarify like so:

```suggestion
export const schuldenType = z.enum([
  "privat",
  "behoerden",
  "kredit",
  "krankenkasse",
  "rechnung",
  "beitragsservice",
  "unterhalt",
  "strafe",
  "finanzamt",
  "hauptzollamt",
  "nichtSagen",
  "weissNicht",
]);
```